app/black_spots/models.py

```diff
@@ -31,7 +34,8 @@ class BlackSpotSet(AshlarModel):
     #: DateTime when the black spots become effective
     effective_start = models.DateTimeField()
 
-    #: DateTime when the black spotd are no longer effective. Should be null when first inserted.
+    #: DateTime when the black spots are no longer effective.
+    #  Should be null when first inserted.
```

Comment:
We've decided on max 100 columns per line, so these comments don't need to be broken up. May want to update your editor defaults.